plugins/modules/inventory_intent.py

```diff
@@ -707,7 +709,8 @@ def get_diff_merged(self, config):
 
             except Exception as e:
                 self.log("An error occurred while Resyncing device in Cisco DNA Center")
-                raise Exception(f"'Error while Resyncing device in Cisco DNA Center' - {str(e)}")
+                error_message = "Error while Resyncing device in Cisco DNA Center - {0}".format(str(e))
```

Comment:
can we use this error_message string in self.log as well?

Reply:
yes, we can